### ISSUE TITLE
Feature request: Local copy of vault

### ISSUE THREAD

**Issue by morgankevinj** [nextcloud/passman-webextension#306]
The extension should store a local copy of the password vault encrypted using the master password. This would allow the user to still use the extension in the case of a temporary loss of connectivity to the server.

**Comment by nicman23** [comment]
Yes please. In run passman on a arm5 800mhz nas and this would up the ux by 2 minutes because that is how long it takes :/

**Comment by binsky08** [comment]
This project has been completely rewritten (v3).
The old code is archived in the `legacy-v2` branch.
Please check whether the issue still exists in the current v3, and if so, please report it again.

It's default behaviour in v3.
